src/main/java/nl/tudelft/ti2206/group9/gui/MainMenuScene.java

```diff
@@ -94,32 +95,34 @@ private void setButtonFunction(final Button button,
 			final BType type) {
 		button.setOnAction(event -> {
 			SplashScene.getButtonAudioPlayer().play(false);
-				if (type == BType.EXIT) {
-					apMainMenu.stop();
-					OBSERVABLE.notify(Category.MENU, Menu.EXIT);
-					ShaftEscape.exit();
-				} else if (type == BType.START) {
-					apMainMenu.stop();
-					if (checkPlayerName(INPUT.getText())) {
-						createNewGame();
-					} else {
-						setPopup(new WarningPopup(
-								event1 -> setPopup(null),
-								"The given name is invalid."));
-						ShaftEscape.showPopup(getPopup());
-					}
-				} else if (type == BType.LOAD) {
-					OBSERVABLE.notify(Category.MENU, Menu.LOAD_MENU);
-					ShaftEscape.setScene(new LoadGameScene());
+			if (type == BType.EXIT) {
+				apMainMenu.stop();
+				OBSERVABLE.notify(Category.MENU, Menu.EXIT);
+				ShaftEscape.exit();
+			} else if (type == BType.START) {
+				apMainMenu.stop();
+				if (checkPlayerName(INPUT.getText())) {
+					createNewGame();
 				} else {
-					OBSERVABLE.notify(Category.MENU, Menu.SETTINGS);
-					ShaftEscape.setScene(new SettingsScene());
+					setPopup(new WarningPopup(
+							event1 -> setPopup(null),
+							"The given name is invalid."));
+					ShaftEscape.showPopup(getPopup());
+				}
+			} else if (type == BType.LOAD) {
+				OBSERVABLE.notify(Category.MENU, Menu.LOAD_MENU);
+				ShaftEscape.setScene(new LoadGameScene());
+			} else if (type == BType.SHOP) {
+				OBSERVABLE.notify(Category.MENU, Menu.SHOP);
+				ShaftEscape.setScene(new ShopScene());
+			} else {
+				OBSERVABLE.notify(Category.MENU, Menu.SETTINGS);
+				ShaftEscape.setScene(new SettingsScene());
 			}
 		});
 	}
 
 	/**
-<<<<<<< HEAD
 	 * Checks whether the playername is a valid name.
 	 * Invalid options:
 	 * <a>
```

Comment:
Guys, please pay more attention with merging. This <a> tag had been changed to <ul> in Mathias' branch, but it's been reverted. Please fix.

<documentation,review